docs/src/Switch.doc.js

```diff
@@ -9,11 +9,37 @@ import PageHeader from './components/PageHeader.js';
 const cards = [];
 const card = c => cards.push(c);
 
+const toggleRTL = () => {
+  if (document && document.documentElement) {
+    const isRTL = document.documentElement.dir === 'rtl';
+    document.documentElement.dir = isRTL ? 'ltr' : 'rtl';
+  }
+};
+
+const changeLocaleDirection = (
```

Comment:
@selena-zeng Could we add this to the navbar instead of making it specific to `Switch`?

Reply:
changed to nav bar